### DISCUSSION TITLE
Temporal consistency of observations

### DISCUSSION THREAD

**Discussion by mankoff** [IACS-cryo/Delineation-WG#23] [Observations]
The most challenging issue raised today (by @willkochtitzky although many contributed to the discussion) is **temporal consistency**.

At the moment, it appears that
+ RGI 7 is Y2000
+ Antarctic ice sheet: Radarsat has a Y2000 coastline product (suggested by @celiaamelie)
+ Greenland ice sheet: PROMICE will have a Y2022 product (from @nielsjk; to be available soon)
  + The PROMICE product will include all connected ice mass, even if weakly connected (e.g., neither dynamics nor hydraulics)

Converting the PROMICE Y2022 product to Y2020 will take "a few months". Converting it to Y2000 will probably take significantly longer. Therefore, temporal consistency is a nice goal, but may not be achievable within reasonable effort, and we will probably need to work with what we have even if different products cover different time periods.

This may not be a problem for **spatial overlap**. We can take a Y2022 ice sheet, a Y2000 peripheral (from RGI), and as long as there is no spatial overlap (verifiable with a GIS query), we avoid double counting.

However, this does introduce a risk of undercounting: If some ice has become disconnected between 2020 and 2022, that ice will not be in the 2022 PROMICE Greenland ice sheet connected product, but it may also not be included in the 2000 RGI product because it appeared part of the main ice at that point in time. These areas are likely to be small.

If you have thoughts about temporal consistency, please reply (remove text if replying via email) on things that I missed or mis-characterized, and, if at all possible, provide suggestions for work-arounds other than re-doing the PROMICE product for Y2K.

**Comment by nielsjk** [comment]
To be precise: I expect completion of GIS 2022 mid-summer, but depends on resources. A revision of the margin to match a 2020 position is probably 4 months full time and not including a revision of nunataks.

A solution could be to also map all CL1 glaciers that has become detached up to 2022.

A coastline will also be part of the data - it is used to create a gridded 2022 ice/ocean/land mask.

**Comment by willkochtitzky** [comment]
Thanks for the meeting yesterday Ken and everyone. First, I am really looking forward to seeing the amazing work Neils and his team are doing, it sounds awesome. However, I think it is a mistake to rely on products that are not yet out. I don't understand the available ice sheets masks as well as others, but I think we should use the existing datasets in our work here. These have been in the community for some number of years and we have a sense of how they are used and people know them. The only thing I know for sure is that we will get this wrong, we will not have the correct partitioning of the ice sheet and periphery glaciers, but I don't think that matters. It is far more important for the community to agree on the partitioning than for the partitioning to be correct.

And saying that RGI is 2000 is just aspirational. Those outlines can be from any year, but the target date is 2000. We made a massive leap forward in reaching that target year with RGI7, but it is still an aspiration and some outlines (although few) are decades away from 2000. I also don't think this matters because it is far more important for the community to agree on the partitioning than for the partitioning to be correct.
